From: Debian Cyrus SASL Team
 <pkg-cyrus-sasl2-debian-devel@lists.alioth.debian.org>
Date: Thu, 24 Mar 2016 11:35:03 +0100
Subject: Include dbconverter-2 in sbin_PROGRAMS and set default sasldb file
 to /etc/sasldb2

database file to /etc/sasldb2.
---
 utils/Makefile.am     | 4 ++--
 utils/dbconverter-2.c | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/utils/Makefile.am b/utils/Makefile.am
index 6ffd8b9..2dcafca 100644
--- a/utils/Makefile.am
+++ b/utils/Makefile.am
@@ -45,10 +45,10 @@
 all_sasl_libs = ../lib/libsasl2.la $(SASL_DB_LIB) $(LIB_SOCKET)
 all_sasl_static_libs = ../lib/.libs/libsasl2.a $(SASL_DB_LIB) $(LIB_SOCKET) $(GSSAPIBASE_LIBS) $(GSSAPI_LIBS) $(SASL_KRB_LIB) $(LIB_DES) $(PLAIN_LIBS) $(SRP_LIBS) $(LIB_MYSQL) $(LIB_PGSQL) $(LIB_SQLITE)
 
-sbin_PROGRAMS = @SASL_DB_UTILS@ @SMTPTEST_PROGRAM@ pluginviewer
+sbin_PROGRAMS = @SASL_DB_UTILS@ @SMTPTEST_PROGRAM@ pluginviewer dbconverter-2
 EXTRA_PROGRAMS = saslpasswd2 sasldblistusers2 testsuite testsuitestatic smtptest pluginviewer
 
-noinst_PROGRAMS = dbconverter-2
+#noinst_PROGRAMS = dbconverter-2
 
 if NO_SASL_DB_MANS
 man_MANS = 
diff --git a/utils/dbconverter-2.c b/utils/dbconverter-2.c
index 4e1bc13..9de1a87 100644
--- a/utils/dbconverter-2.c
+++ b/utils/dbconverter-2.c
@@ -385,7 +385,7 @@ static struct sasl_callback goodsasl_cb[] = {
 
 int main(int argc, char **argv)
 {
-    const char *db="/etc/sasldb";
+    const char *db="/etc/sasldb2";
     int result;
 
     if (argc > 1) {
@@ -421,7 +421,7 @@ int main(int argc, char **argv)
     
     printf("\nThis program will take the sasldb file specified on the\n"
            "command line and convert it to a new sasldb file in the default\n"
-           "location (usually /etc/sasldb). It is STRONGLY RECOMMENDED that you\n"
+           "location (usually /etc/sasldb2). It is STRONGLY RECOMMENDED that you\n"
            "backup sasldb before allowing this program to run\n\n"
 	   "We are going to convert %s and our output will be in %s\n\n"
            "Press return to continue\n", db, db_new);
